src/tk_scrollable_frame.py

```diff
@@ -0,0 +1,104 @@
+import tkinter as tk
+import platform
+
+
+# ************************
+# Scrollable Frame Class
+# ************************
+class ScrollFrame(tk.Frame):
+    """
+    This is based on the code from this GIST: ttps://gist.github.com/mp035/9f2027c3ef9172264532fcd6262f3b01
```

Comment:
I like this change, but unfortunately this GIST isn't licensed, so it would be unlawful for us to re-distribute it with the GPL license that the OpenMCR code is available under.

1. Is this change _required_ to make the program run on MacOS? Could we instead pull it out to a new PR that specifically addresses this UI problem? I think this issue tracks this problem: https://github.com/iansan5653/open-mcr/issues/16
2. When we do fix this, is there a public library available with a compatible license that we could use instead?

Reply:
Oh never mind to the first item in that list! I see you already addressed it in https://github.com/iansan5653/open-mcr/pull/74. Could we just remove these changes from this PR and from https://github.com/iansan5653/open-mcr/pull/76?